bin/_inserts

```diff
@@ -33,6 +35,7 @@ insert_file_over_pattern 'config/application.rb' 'Bundler.require'
 # One important part of this is exercising the JS/CSS assets.
 #
 cp ../inserts/config/routes.rb config/routes.rb
+mkdir -p app/views/layouts
```

Comment:
Another file safety so the bin/build runs to completion. Technically the view files below are not needed.